### ISSUE TITLE
Unable to update from Python 3.12.0 to 3.12.1 using your repositories in Ubuntu 22.04

### ISSUE THREAD

**Issue by dilnix** [deadsnakes/issues#272]
### description

There is a problem updating Python 3.12 from 3.12.0 to 3.12.1 using your repositories in Ubuntu 22.04.
Unable to update packages `libpython3.12-testsuite` and `python3.12-full`.
The specific syntax error occurs while trying to update the package `libpython3.12-testsuite`:
```
Setting up libpython3.12-testsuite (3.12.1-1+jammy1) ...
  File "/usr/lib/python3.12/test/test_future_stmt/badsyntax_future10.py", line 3
    from __future__ import print_function
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: from __future__ imports must occur at the beginning of the file
```
Which is the dependency problem to install `python3.12-full`.

![Screenshot_20231209_211434](https://github.com/deadsnakes/issues/assets/33361500/d3e9ba17-053b-4a93-9732-f05ecdd4eab8)

#### lsb_release -a

```
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy
```

#### uname -a

```
Linux 5CD116MK0C 5.19.0-50-generic #50-Ubuntu SMP PREEMPT_DYNAMIC Mon Jul 10 18:24:29 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

Please provide the fixed packages via the repository.

**Comment by asottile** [comment]
ah the same problem as #265 but now in 3.12

**Comment by dilnix** [comment]
Yes, I have seen. It's not an urgent problem. But creating an issue with a hope that this will grab some attention and the rebuilt package will be delivered sooner. =]

**Comment by asottile** [comment]
jammy2 and focal1 should have this fix -- they'll be built and available once launchpad is done!

**Comment by dilnix** [comment]
I don't if it's a joke, but this afternoon (EET) I see the another issue with installing the updated `libpython3.12-testsuite` package. =)

![Screenshot_20231210_135443](https://github.com/deadsnakes/issues/assets/33361500/1fea2c93-3328-4e94-b893-a458fec796a1)

This time, the syntax error is about old style print usage...
```
Setting up libpython3.12-testsuite (3.12.1-1+jammy2) ...
  File "/usr/lib/python3.12/test/test_lib2to3/data/bom.py", line 2
    print "BOM BOOM!"
    ^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```

**Comment by dilnix** [comment]
@asottile, do you know what is going on? 👆

**Comment by asottile** [comment]
don't ping me -- look at #273
